README.md

```diff
@@ -62,13 +62,14 @@ npm install -g esy@0.5.7
 - `esy install`
 - `esy '@test' install`
 
-### Building
+### Building from Head
 
 - `esy build`
 
-### Running tests
+### Running tests in a testing environment
```

Comment:
Again I'm not sure what "testing environment" means. Are there environments where you cannot run the tests like this?

Reply:
same as here: https://github.com/onivim/libvim/pull/178#discussion_r383946457 

Basically, I'm interested in removing code from libvim, so my workflow is really:

- remove code
- run tests
- commit

I never actually have to `esy build` in that flow. I imagine others will want to contribute in this manner for the forseeable future so was trying to figure out how to word that :)